src/main/java/org/amahi/anywhere/server/api/ServerApi.java

```diff
@@ -46,32 +46,27 @@
 public interface ServerApi {
     @POST("/auth")
     Call<HdaAuthResponse> authenticate(
-        @Header("Session") String session,
```

Comment:
Why did you remove the session header? The remote browsing has stopped working.
The session token is for the pfe to identify which fs to connect to (so its not needed in local mode but required in remote mode) where as the auth token is for the fs to identify which hda-user to serve the content to.

Reply:
You may keep the session header as optional, when you have it in case of remote browsing, send it. When in local mode (using pin access), don't send it as you don't have it.

Reply:
ok, I will restore the session code. For local pin access what there is no remote address, Should I hide remote connection from settings for it or set `remote address = local address`?

Reply:
For now hide/disable the remote option (if its not too difficult to do so  or else make equal). In future we have to implement remote access for NAU also, so mark a TODO in code where you make this change.